### PR TITLE
Blogging in HTML

### PR DESCRIPTION
**Pull request by morgangant16**

### CODE REVIEW

**Comment by JacobH93** [comment]
Hey Morgan, I haven't used this feature before so hopefully, you can see the file. I am attaching a txt file with some proposed changes to "add files via upload page". It is possible that it is just my browser but I don't see anything within your code tags when I load the page. I have found that inserting & l t ; and & g t ; wherever there are less than or greater than signs fixes this problem. I also added in some breaks to make things easier to read. Please let me know what you think! thanks
[pull.txt](https://github.com/gvsoftwaredev2021/gvsoftwaredev2021/files/7141321/pull.txt)

**Comment by JacobH93** [comment]
Alternatively, using &lt;xmp&gt; whatever tags you want to show as plain text &lt;xmp&gt; works.. which would've been way faster than what I did for the first example. haha

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# gvsoftwaredev2021
\ No newline at end of file
+<!DOCTYPE html>
```
Comment (ogennadi) [inline README.md:1]: Please move your logs to the main file

File: Blogging in html.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (ogennadi) [inline Blogging in html.html:1]: Please make the file name all lowercase and following the pattern in the assignment statement

**Comment by Evan-Callaghan** [comment]
Good work Morgan and David. There are a lot of different documents going on here so it is a little difficult to follow along.

**Pull request closed without merging**